src/net.cpp

```diff
@@ -81,6 +81,9 @@ struct ListenSocket
 };
 }
 
+/** Services this node implementation cares about */
+static const uint64_t nRelevantServices = NODE_NETWORK;
```

Comment:
Thinking about this a little more I think it would be wiser to no disallow outbound nodes that are not NODE_NETWORK, but rather limit their number (or prune them if there are too many, perhaps allow 2 out of the 8 outbounds max to be non NODE_NETWORK?).  In this way we can still connect to non NODE_NETWORK nodes and share addresses while keeping the greater XTHIN network well connected.  The issue I have with this commit is that it will slowly isolate pruned nodes at a time when pruned nodes are rising in number (because the blockchain is growing) and also at a critical time in BU where we may have a possible upcoming UASF fork to contend with and possible network wide sybil attacks.